test/dir-mixin.test.js

```diff
@@ -0,0 +1,97 @@
+import { expect } from '@esm-bundle/chai';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { DirMixin } from '../vaadin-dir-mixin.js';
+
+class DitMixinElement extends DirMixin(PolymerElement) {
```

Comment:
`DitMixinElement` is this a typo?
It's for a test so not a blocker

Reply:
Thanks, renamed.